modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,196 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
```

Comment:
CONTEXT variables are usually placed in Assemblies and not modules. Is there a specific reason for placing a context variable in a module?

Reply:
Agreed. I'll remove it. Was following the format for other service mesh modules.